scripts/systemtap/kem/kemd.c

```diff
@@ -264,16 +262,16 @@ static int kemd_release(struct inode *inode, struct file *filp)
 }
 
 static ssize_t kemd_read(struct file *filp, char *buf,
-                         size_t len, loff_t *offset)
+						 size_t len, loff_t *offset)
 {
-	int             bytes_read = 0;
+	int bytes_read = 0;
 	struct kem_dev *kemd_dev;
-	struct kem_rb  *rb;
-	struct ke_msg  *ent;
-	unsigned int    nr_ents;
-	unsigned int    nr_written;
-	int             i;
-	int             err;
+	struct kem_rb *rb;
+	struct ke_msg *ent;
+	unsigned int nr_ents;
+	unsigned int nr_written;
+	int i;
+	int err;
```

Comment:
Please restore to the original indentation and alignment.

Reply:
restored